test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -572,6 +573,13 @@ spec =
                     v -> Test.fail "Expecting Integer, but got: "+(Meta.type_of v).to_text
                 sum.div two_hundred.length . should_equal expected_value
 
+            Test.specify name+" BigInteger to BigDecimal test" <|
+                    h = 2^70
+                    bd1 = BigDecimal.new h 0
+                    bd2 = BigDecimal.new h
+                    bd1.to_text . should_equal bd2.to_text
+                    bd1 . should_equal bd2
```

Comment:
```suggestion
                    bd1 . should_equal bd2
                    bd1.toBigIntegerExact . should_equal h
```

I'd add this just to be sure.

I assume you did check the value of `BigDecimal.new h 0` to be correct, but without checking it explicitly there is no guarantee that _both_ of these big decimals will be e.g. `0`. As this could potentially regress when other changes to polyglot dispatch are done, I think good to have a 'sure' check - which `bd1.toBigIntegerExact . should_equal h` is because it compares with the original 'reference' value.